java/api/pom_template.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>io.ray</groupId>
     <artifactId>ray-superpom</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.5.0</version>
```

Comment:
why are these changed to 2.5.0 ? why 2.5.0?

Reply:
ouch I must forget to change the version in the second command.